### DISCUSSION TITLE
Weekly Office Hours - 2024-06-04

### DISCUSSION THREAD

**Discussion by jonesbusy** [jenkins-infra/plugin-modernizer-tool#13] [Meeting notes]
Done

Action point

Availability

**Comment by sridamul** [comment]
Note to new viewer: Meeting notes are recorded in the below docs: https://docs.google.com/document/d/1AWv6aEL9NU5hVW_CJl81ethsguVqqL2e7H0PYOUXnR8/edit?usp=sharing
